lib/charms/vault_k8s/v0/vault_managers.py

```diff
@@ -0,0 +1,372 @@
+"""Library for managing Vault Charm features.
+
+This library encapsulates the business logic for managing the Vault service and
+its associated integrations within the context of our charms.
+"""
+
+# The unique Charmhub library identifier, never change it
```

Comment:
This comment should be next to the lib id?

Reply:
:D -- auto imports :)

oops

Reply:
https://github.com/canonical/vault-k8s-operator/pull/534/commits/1ab346f30cc700a7d01eee6d821e2757552fc977